OpenMEEGMaths/src/MathsIO.C

```diff
@@ -23,7 +23,7 @@ namespace OpenMEEG {
                 for(int i=maxtagsize-1;i>=0;--i)
                     is.putback(buffer[i]);
 
-                return buffer;
+                return std::string(buffer, maxtagsize);
```

Comment:
En utilisant le flag "-fsanitize=address" , comme `read` ne met pas de NULL-character, il faut préciser la taille pour ne pas faire de `heap-buffer-overflow`

Reply:
Pour la mkl, je pense que passer par les depot intel sera plus efficace (et plus a jour). Je l'avais fait mais Alex est je crois revenu au deport conda de l'ancienne version....

Reply:
Pour le chanegement de MathsIO.h, je regaedw, mais a priori, ca semble raisonnable.

Reply:
@papadop you mean use the conda intel channel to get mkl? if so it means you require to have conda to build with mkl which is maybe not great.

Reply:
No I mean the use of their yum and deb repositories, which I use on my laptop, since january when I discovered their existence and tried them in travis.

Reply:
https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-yum-repo
J'ai vu aussi ce lien, mais rien pour MacOs si ?